### ISSUE TITLE
Website: use aria-current to indicate current nav

### ISSUE THREAD

**Issue by strager** [quick-lint/quick-lint-js#1024]
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-current

**Comment by devinhadley** [comment]
I would like to give this a shot if possible.

**Comment by devinhadley** [comment]
My manual testing has indicated that the aria-current="page" is being correctly defined for the current page. However, im struggling to find a way to actually test the implementation using accessibility software. Windows narrator (and chrome Screen Reader) don't seem to treat aria-current="page" specially. However, if I use aria-label it does correctly read whatever the label is for just the current page when it is hovered over.

The fact label is working, and aria-current="page" is valid according to the [docs](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-current) i'm not really sure how to proceed with testing. I also tested changing the spans to <a> with an href and still no dice. 

How should I proceed with testing?
